.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "FireRender.Components"]
 	path = RadeonProRenderSharedComponents
-	url = git@github.com:GPUOpen-LibrariesAndSDKs/RadeonProRenderSharedComponents.git
+	url = git@github.com:DMotorygin/RadeonProRenderSharedComponents.git
```

Comment:
Repository is changed. Is it expected?

Reply:
can you just make a PR to shared components that I can merge?